tests/scripts/t3000/run_t3000_demo_tests.sh

```diff
@@ -13,7 +13,7 @@ run_t3000_falcon40b_tests() {
   WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -n auto models/demos/t3000/falcon40b/tests/ci/test_falcon_end_to_end_60_layer_t3000_prefill_10_loops.py --timeout=720 ; fail+=$?
 
   # Falcon40B end to end demo (prefill + decode)
-  WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -n auto models/demos/t3000/falcon40b/tests/ci/test_falcon_end_to_end_t3000_demo_loops.py ; fail+=$?
+  WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -n auto models/demos/t3000/falcon40b/tests/test_demo.py::test_demo ; fail+=$?
```

Comment:
Can we invoke just the test_demo.py file itself?

It keeps workflows cleaner and easier to debug for pipeline maintenance
there is also an issue about this https://github.com/tenstorrent/tt-metal/issues/7586

Reply:
Invoking `test_demo.py` now and skipping the test to generate the reference outputs using `is_ci_env` fixture 👍